src/reply.rs

```diff
@@ -427,6 +427,23 @@ where
     }
 }
 
+impl<T> Reply for Result<T, std::convert::Infallible>
+where
+    T: Reply + Send,
+{
+    #[inline]
+    fn into_response(self) -> Response {
+        match self {
+            Ok(t) => t.into_response(),
+            Err(e) => {
+                // Maybe panic?
+                tracing::error!("Infallible Error: {:?}", e);
+                StatusCode::INTERNAL_SERVER_ERROR.into_response()
+            }
```

Comment:
If I understand correctly, this should work better
```suggestion
            Err(e) => match e {}
```

Reply:
I did not know that about `Infallible`, I changed it to what you suggested.